### DISCUSSION TITLE
Skip  Dislikes

### DISCUSSION THREAD

**Discussion by mavorse** [pear-devs/pear-desktop#829] [Ideas]
Hi there, great projekt, thank you. 
Most wanted feature, skip music I don't like,  there is a plugin 
I suggest integrate skip dislike plugin,  https://chrome.google.com/webstore/detail/skip-youtube-music-dislik/iggmfincgcgnckljenalmhilapdeeina

**Comment by VanGeko** [comment]
yes, pls!

**Comment by VanGeko** [comment]
any progress for this?

**Comment by DereC4** [comment]
interesting

**Comment by TecnikOfficial** [comment]
> Hi there, great projekt, thank you. Most wanted feature, skip music I don't like, there is a plugin I suggest integrate skip dislike plugin, https://chrome.google.com/webstore/detail/skip-youtube-music-dislik/iggmfincgcgnckljenalmhilapdeeina

the plugin u shared is not open source plus its last updated in 2021 unknown dev plus not much ratings to back its trustworthiness and not all " featured " extensions r secured so adding this will be a security risk as there is google login  involved in yt music so thaking tht chance is not a good option 
watch this video by ThioJoe on browser extensions for more info https://youtu.be/Al0K-bnC2Yw

**Comment by inson1** [comment]
@mavorse What about closing this, now when the plugin was added?

**Comment by VanGeko** [comment]
its no longer working, since last update,
when pressing dislike on a song, it skips the song, but disliked songs who are already in a list, still played.

**Comment by VanGeko** [comment]
![no dislike](https://github.com/user-attachments/assets/fcf5244f-25d1-4f12-9837-892390d90f5a)

still not working.
